tests/Projekt.Tests/Projekt.Tests.fsproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think this file has changed meaningfully, could you revert the changes please?